tests/setup/globalSetup.js

```
@@ -40,5 +51,88 @@ module.exports = async () => {
+  // Match the codebase auth convention from src/utils/fetch.ts:
+  //   OAuth mode → Authorization: Bearer <token>
+  //   PAT mode   → PRIVATE-TOKEN: <token>  (GitLab's canonical PAT header)
+  // PATs also accept Bearer, but PRIVATE-TOKEN keeps us consistent with the
+  // rest of the codebase and avoids confusion when debugging auth issues.
+  // AbortController guards against a hung connection blocking suite startup
+  // (Jest's per-test timeout doesn't apply in globalSetup).
+  const oauthMode = String(process.env.OAUTH_ENABLED ?? '').toLowerCase() === 'true';
+  const authHeaders = oauthMode
+    ? { Authorization: `Bearer ${process.env.GITLAB_TOKEN}` }
+    : { 'PRIVATE-TOKEN': process.env.GITLAB_TOKEN };
+  const controller = new AbortController();
```

ESLint

[error] 'AbortController' is not defined.

(no-undef)